### DISCUSSION TITLE
dokku and managed mongo

### DISCUSSION THREAD

**Discussion by coaperator** [dokku/dokku#4960] [Q&A] [answered]
guys help pliz, I'm trying to use this
https://www.digitalocean.com/products/managed-databases-mongodb/
according to this instruction
https://docs.digitalocean.com/products/databases/mongodb/how-to/connect/

but I have an application in a container, and it receives a connection string via ENV (Dokku installs at the build stage)
so here is the problem with how to pass the correct path to the certificate file
like that
`dokku config:set my_app MONGO_URL="mongodb+srv://mongodb:pass@path_to_server?authSource=admin&replicaSet=dev-mongo&tls=true&tlsCAFile=/root/dev-mongo-ca-certificate.crt"`
the application is in the container and naturally does not see the file (it is on the host machine)
not shove the certificate into the container? what to think of?

**Comment by josegonzalez** [comment, marked as answer]
You can mount the certificate directory via `storage:mount` and then reference it in the container path.
